src/client/src/system/openFin/openFin.ts

```diff
@@ -47,7 +46,7 @@ export default class OpenFin {
           currentWindow.restore(() => currentWindow.bringToFront(
             () => log.info('Window brought to front.'),
             err => log.error(err),
-          ), err => log.error(err))
+          ), () => log.error())
```

Comment:
why did you remove the error?

Reply:
Does not exist according to TS definitions

Reply:
Fixed

Reply:
It does according to the docs http://cdn.openfin.co/jsdocs/stable/fin.desktop.Window.html#restore